src/main/scala/edu/ie3/simona/service/weather/WeatherSource.scala

```diff
@@ -559,7 +559,7 @@ object WeatherSource {
     */
   object WeatherScheme extends ParsableEnumeration {
     val ICON: Value = Value("icon")
-    val PSDM: Value = Value("psdm")
+    val Cosmo: Value = Value("psdm")
```

Comment:
Please rename to uppercase `COSMO` and also change the String parameter to `"cosmo"`

Reply:
Done